api/index.ts

```diff
@@ -11,11 +11,11 @@ const app = express()
 
 const passportInitializer = container.resolve(PassportInitializer)
 passportInitializer.initialize()
-passportInitializer.install(app)
+const redis = passportInitializer.install(app)
 
 export const server = new ApolloServer({
   schema: loadSchema(),
-  context: buildContext,
+  context: ({ req, res }) => buildContext({ req, res, redis }),
```

Comment:
I couldn't find any way to pass this redis client to buildcontext function to access it inside resolvers. So For the time being I hard-coded in the node-module. This function is from graphql-passport (node_modules/graphql-passport/lib/buildContext.d.ts). I don't think it the correct way to do this.